ExtractUserDataActivity/handler.ts

```diff
@@ -463,27 +470,56 @@ export const queryAllUserData = (
       })
     ),
     // step 3: queries notifications statuses
-    TE.chain(
+    TE.bind("notificationStatuses", ({ notifications }) =>
+      findAllNotificationStatuses(notificationStatusModel, notifications)
+    ),
+    // step 4: queries profile service preferences
+    TE.bindW("servicesPreferences", () =>
```

Comment:
the left part is the same of the upper chain, `W` is not needed.
```suggestion
    TE.bind("servicesPreferences", () =>
```

Reply:
nice catch, done in https://github.com/pagopa/io-functions-admin/pull/191/commits/b8fc8ca5c09962851f6e1e5838421523e7643955